controller/ResultTable.php

```diff
@@ -68,7 +68,7 @@ public function index()
         $deliveryService = DeliveryAssemblyService::singleton();
         if($this->getRequestParameter('classUri') !== $deliveryService->getRootClass()->getUri()) {
             $filter = $this->getRequestParameter('filter');
-            $classUri = $this->getRequestParameter('classUri');
+            $classUri = $this->getRequestParameter('uri');
```

Comment:
If the `uri` is used instead of the `classUri`, please update the variable name. Then there's a call to `$this->setData('classUri', $classUri);` which will contain the uri. This is going to be confusing.
Could you please update the whole chain.